merlin/systems/dag/ensemble.py

```diff
@@ -116,17 +119,34 @@ def export(self, export_path, version=1):
                     model_name=node_name, model_version=-1
                 )
 
-                for input_col_name in node.input_schema.column_names:
+                for input_col_name, input_col_schema in node.input_schema.column_schemas.items():
                     source = _find_column_source(node.parents_with_dependencies, input_col_name)
                     source_id = node_id_lookup.get(source, None)
                     in_suffix = f"_{source_id}" if source_id is not None else ""
-                    config_step.input_map[input_col_name] = input_col_name + in_suffix
-
-                for output_col_name in node.output_schema.column_names:
+                    if input_col_schema.is_list and input_col_schema.is_ragged:
+                        config_step.input_map[input_col_name + "__values"] = (
```

Comment:
I'd really like to create some abstraction to represent both of the arrays that represent ragged multi-hot columns, but we're not there yet so we have to handle this case explicitly in several places.